src/wp-includes/functions.php

```diff
@@ -4822,7 +4852,7 @@ function wp_parse_args( $args, $defaults = array() ) {
 	if ( is_object( $args ) ) {
 		$parsed_args = get_object_vars( $args );
 	} elseif ( is_array( $args ) ) {
-		$parsed_args =& $args;
+		$parsed_args = & $args;
```

Comment:
```suggestion
		$parsed_args =& $args;
```
This was an unrelated change that PHPCBF seems to have made. :slightly_smiling_face: